### DISCUSSION TITLE
RaspberryMatic über HA: Auswahl Wochenprogramm wird nicht übernommen

### DISCUSSION THREAD

**Discussion by loeh72** [OpenCCU/OpenCCU#2951] [🇩🇪 Fragen&Antworten] [answered]
Hallo, bin leider kein Intensivanwender von HA und RaspberryMatic. Ich habe seit ich RaspberryMatic über HA nutze folgendes Problem.
Für meine Heizkörpersteuerung über HmIP-RF habe ich Wochenprogramme. Jedes Mal wenn ich versuche das Programm von z.B. 1 auf Programm 2 zu stellen wird es nicht übernommen und bleibt auf 1 stehen.  Ändere ich aber z.B. minimale Temp. oder max. Temperatur wird dies an den Aktor übertragen. Eingesetzte Vers. HA OS 14.0; Core 2024.12.1;  RaspberryMatic 3.79.6.20241122
Um Unterstützung wäre ich Dankbar.

**Comment by jens-maus** [comment]
Ein Bild und/oder Video sagt mehr als tausend Worte...

**Comment by loeh72** [comment]
Ich glaube ich habe verstanden. Siehe File.

https://github.com/user-attachments/assets/64d6783c-ac39-4d1d-95ae-ce1a5fa4097a

**Reply by jens-maus** [reply, marked as answer]
Das ist der falsche Ort um das Heizkörperventil von einem Wochenprogram (wie gezeigt von 1 auf 3) zum anderen zu stellen. Das muss unter "Status und Bedienung -> Geräte" passieren. Dort stellt man von einem Wochenprogram zum anderen um. An der Stelle wo du das versuchst konfiguriert man lediglich die Wochenprogrammzeiten.

**Reply by loeh72** [reply]
Oh Mann, also wie so oft saß das Problem vor dem Bildschirm :-), Vielen Dank für deine Info und Hilfe. Kaum macht man es richtig funktioniert es auch!
